src/Behavioral.Automation/Services/PrintValuesHelper.cs

```diff
@@ -8,7 +8,8 @@ public static class PrintValuesHelper
     {
         public static string GetPrintableValues(this IEnumerable<ITableRowWrapper> rows)
         {
-            return rows.SelectMany(x => x.CellsText).Aggregate((x, y) => $"{x}, {y}");
+            var values = rows.Aggregate("\r\n|", (current, row) => current + (row.CellsText.Aggregate((x, y) => $" {x} | {y}") + " |\r\n |"));
+            return values.Remove(values.Length - 3);
```

Comment:
Why do we need to remove the last three characters from `values`?

Reply:
It's the last letters with "\r\n |"